vspace/src/main/webapp/WEB-INF/views/staff/spaces/customorder/add.html

```diff
@@ -0,0 +1,157 @@
+<html layout:decorate="~{layouts/main_staff}">
+	<head>
+		<style>
+			.addNewCustomOrder {
+				background: white;
+				padding: 30px 40px;
+			}
+			.orderName, .orderDescription {
+				border: 1px solid;
+			}
+			.dragDropText {
+				margin-bottom: 10px;
+			}
+			.sortableSpaces {
+				max-height: 205px;
+				overflow-y: scroll;
+			}
+			.paraContent {
+				cursor: pointer;
+				border: 1px solid;
+				padding: 15px;
+			}
+			.paraContent:hover {
+				background: #D3D3D3;
+				font-weight: bold;
+			}
+			.para {
+				margin: 0;
+			}
+			.errorMsgFailed {
+				position: absolute;
+    			top: 8%;
+    			width: 65%;
+    			display:none;
+			}
+			.goBackContainer {
+				float: right;
+			}
+			.goBackContainer:hover {
+				cursor: pointer;
+				font-weight: bold;
+			}
+		</style>
+		<link th:href="@{/resources/multiselect/css/multiselect.css}" type="text/css" rel="stylesheet">		    
+		<script th:src="@{/resources/multiselect/js/multiselect.min.js}"></script>
+		<script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.3/jquery-ui.min.js"></script>
+		
+		<script th:inline="javascript">
+		var spaceDivs = []
+		$(document).ready(function() {
+			$("#errorMsgFailed").hide();
+			var vals = []
+			
+			spaceDivs = document.getElementById('sortableSpaces').children;
+			var spaceIds = [];
+			let index =0;
+			 for(ele of spaceDivs){
+				spaceIds.push(spaceDivs[index].children[0].id);
+				index++;
+			} 
+			 spaceDivs=[];
+			 spaceDivs = spaceIds;
+			//save order
+			$("#createOrder").click(function() {
+				var formData = new FormData();
+				var name = $("#name").val();
+				var descriptionText = $("#description").val();
+			    formData.append('spaceOrder', spaceDivs);
+			    formData.append('name', name);
+			    formData.append('description', descriptionText);
+			    formData
+			    $.ajax({
+			        url: "[(@{'/staff/space/order/customorder/add'})]"+'?'+[[${_csrf.parameterName}]]+'='+[[${_csrf.token}]],
```

Comment:
csrf should be in body or header; it should only be in the url if it's a file upload